packages/edit-post/src/hooks/components/media-upload/index.js

```diff
@@ -67,7 +67,7 @@ const getAttachmentsCollection = ( ids ) => {
 		order: 'ASC',
 		orderby: 'post__in',
 		post__in: ids,
-		per_page: 100,
+		posts_per_page: -1,
```

Comment:
As noted in my comment, `wp.media.query()` normally uses `perPage` instead of `posts_per_page`, but either should work and I slightly prefer the fact that this is consistent with the WP_Query args that are more familiar.

TL;DR – LGTM.